lib/Template.js

```diff
@@ -81,6 +130,11 @@ module.exports = class Template {
 		);
 	}
 
+	/**
+	 *
+	 * @param {string[] | string} str string to convert to identity
```

Comment:
I'd use `ReadonlyArray<string>|string` instead but that currently breaks the `Array.isArray` check (unless you provide an overload)

Reply:
Do we really want an overload here?